src/room.ts

```diff
@@ -114,7 +122,7 @@ export class Room extends EventEmitter implements Sayable {
       const data = await contactGetter(this.id)
       log.silly('Room', `contactGetter(${this.id}) resolved`)
       this.rawObj = data
-      this.obj    = this.parse(data)
+      this.obj    = await this.parse(data)
```

Comment:
I guess you typo here..
> Also parseNickMap() parseRemarkMap() parseDisplayName() should not be async function too.

you mean parseNickMap() parseRemarkMap() parseDisplayName() should be sync function?